.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pybind11/pybind11"]
```

Comment:
Okay, in that case I would propose waiting to see whether they accept the catkin PR. Otherwise let's make it an add external project and move on